src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminListSearch.cshtml

```diff
@@ -2,32 +2,36 @@
 @inject IDisplayManager<ContentOptionsViewModel> DisplayManager
 @{
     var syntaxThumbnail = await DisplayManager.BuildDisplayAsync(Model, null, "Thumbnail");
+    var dropdownClassList = new List<string>()
+    {
+        "dropdown-menu"
+    };
+
+    if (CultureInfo.CurrentUICulture.IsRightToLeft())
+    {
+        dropdownClassList.Add("dropdown-menu-start");
+    }
 }
-<div class="input-group input-group-sm w-50 d-inline-flex has-filter">
```

Comment:
here try w-md-50 or w-lg-50 instead.

Reply:
@Skrypt are you fine to merge this without this suggestion?

Reply:
@Skrypt if you must have a max width set, we could add `w-xxl-75` just for super large screens.